### PR TITLE
Add ATD Dashboard

### PR DESCRIPTION
**Pull request by antho1404**
A simple version of the ATD dashboard that displays volume and token distributed

Screen Shot 2019-04-24 at 16 30 06

https://mesg-atd.netlify.com/

### CODE REVIEW

**Comment by NicolasMahe** [comment]
Nice :)

can you setup netlify to see preview in real condition?

My feedbacks:
- should it be compatible with multiple transfers per day? Or do you want to only display the total released token (no wallet details)?
- Add a explanation of the ATD system with links to the smart contracts
- Add the circulating supply (so that coinmarketcap can parse it)

**Comment by antho1404** [comment]
live version here https://mesg-atd.netlify.com/
also for the rest, I definitely agree, but this can come later but let's track this in issues

File: frontend/components/distribution/Table.vue [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<template>
+  <v-data-table
+    :headers="headers"
+    :items="items">
+    <template v-slot:items="props">
+      <td>{{ props.item.date | date }}</td>
```
Comment (NicolasMahe) [inline frontend/components/distribution/Table.vue]: Could you output the date in ISO and UTC?

File: frontend/store/token.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import Web3 from 'web3'
+const contracts = require('./atd.json')
+const PRECISION = 1e18
+const web3 = new Web3('https://mainnet.infura.io')
```
Comment (NicolasMahe) [inline frontend/store/token.js]: You should use the infura endpoint with the public api key

File: frontend/store/token.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    const events = (await Promise.all(eventPromises))
+      .reduce((prev, item) => [...prev, ...item], [])
+      .map(event => {
+        return {
+          blockNumber: event.blockNumber,
+          transactionHash: event.transactionHash,
+          date: new Date(event.returnValues.date.toNumber() * 1000),
+          dailyTradedVolume: Math.round(event.returnValues.dailyTradedVolume / PRECISION * 100) / 100,
+          amount: Math.round(event.returnValues.amount / PRECISION * 100) / 100,
+          percentage: Math.round(event.returnValues.amount / event.returnValues.dailyTradedVolume * 100)
```
Comment (NicolasMahe) [inline frontend/store/token.js]: Is it really useful to round the numbers?
The number are already displayed with a specific format:
```
{{ props.item.amount | number('0,0.00') }}
```

**Pull request merged**